apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-card.js

```diff
@@ -54,12 +54,15 @@ function WelcomeTourCard( {
 			is_gutenboarding: isGutenboarding,
 		} );
 	} );
+	const cardMediaClass = classNames( 'welcome-tour-card__media', {
+		'with-extra-padding': isMobile() && imgNeedsPadding,
```

Comment:
Looking at [CSS/Sass Coding Guidelines ](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md), I think it may be correct this way of naming modifiers as well? 🤔  
-- or single underscore are BEM convention, but it seems `.this-is-a-modifier` could also be correct 😄

Reply:
I think this should be fine. It looks right per Calypso conventions - which recommend the form `is-modifier`, etc. I used the double dash a few times myself and plan on refactoring to the Calypso convention.

Reply:
We've briefly discussed this in the previous PR. See https://github.com/Automattic/wp-calypso/pull/59052#discussion_r766753359
I think we agree that BEM modifiers are adding too much verbosity making the JS code harder to read. Then, yes, there are our coding guidelines. I've added the `is-` prefix now to follow them more closely.